eradiate/scenes/tests/test_biosphere.py

```diff
@@ -42,8 +51,8 @@ def test_leaf_cloud_instantiate_parameters(mode_mono):
     # default parameters are used correctly
     cloud2 = LeafCloud.from_parameters(size=[0, 0, 0], leaf_radius=0.25)
     assert cloud2.n_leaves() == 4000
-    max_x = 0,
-    max_y = 0,
+    max_x = (0,)
```

Comment:
why are you turning those into tuples?

Reply:
Black did all by itself because of the commas, I think. I'll fix that.